discord_helpers.py

```diff
@@ -22,3 +27,68 @@ def hasRole(roles, REQUIRED_ROLE_ID):
     if discord.utils.get(roles, id=int(REQUIRED_ROLE_ID)):
         return True
     return False
+
+async def generate_report(conn):
+    last_tx_date = fetch_last_tx()
+
+    if last_tx_date == None:
+        return "Failed to load past transactions of reward account! Does it exist?"
+
+    leaderboard_rows = fetchLeaderboard(conn, last_tx_date, datetime.now())
+    user_rows = fetchUserPubKeys(conn)
+    sumVotes = 0
+
+    BASE_FEE = 10000
```

Comment:
Maybe this can be an env variable or at the top?

Reply:
+1